src/main/java/software/amazon/event/kafkaconnector/EventBridgeWriter.java

```diff
@@ -101,6 +110,28 @@ public EventBridgeWriter(EventBridgeSinkConfig config) {
     this.eventBridgeMapper = new DefaultEventBridgeMapper(config);
     this.batching = new DefaultEventBridgeBatching();
 
+    if ((config.offloadingS3defaultBucket != null) && !config.offloadingS3defaultBucket.isEmpty()) {
```

Comment:
nit: do you want to add some error handling if bucket is set but field ref is not? I know we have a default, but just in case user messes up ;)